qir/stdlib/README.md

```diff
@@ -13,7 +13,7 @@ To manually build the C test,
   mkdir -p target/
 
   # Creating the stdlib LL file
-  clang -O1 -mllvm -disable-llvm-optzns -S -emit-llvm  -c src/stdlib.c -o lib/stdlib.ll
+  Clang -O1 -mllvm -disable-llvm-optzns -S -emit-llvm  -c src/stdlib.c -o lib/stdlib.ll
```

Comment:
Similarly here, is this replacement intended?

Reply:
Thanks! Good catch!